.github/workflows/publish-release.yml

```diff
@@ -192,8 +193,15 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      -
-        name: Install Go
+        
+      - name: Install Zig
```

Comment:
Did you do a test run of the release in GitHub Actions to verify it all works?
I think the way I did this last time I tried was by changing the trigger to include pushes to this branch. And I told GoReleaser to skip version verification and publishing.

Reply:
Yes I did do a test run, which caught a few issues, but now it works.